src/autoscaler/Makefile

```diff
@@ -37,7 +37,9 @@ openapi-generated-clients-and-servers-files = $(wildcard ${openapi-generated-cli
 generate-openapi-generated-clients-and-servers: ${openapi-generated-clients-and-servers-dir} ${openapi-generated-clients-and-servers-files}
 ${openapi-generated-clients-and-servers-dir} ${openapi-generated-clients-and-servers-files} &: $(wildcard ./helpers/apis/generate.go) ${openapi-specs-list} ./go.mod ./go.sum
 	@echo "# Generating OpenAPI clients and servers"
-	go generate ./helpers/apis/generate.go
+	# $(wildcard ./helpers/apis/generate.go) causes the target to always being executed, no matter if file exists or not.
+	# so let's don't fail if file can't be found, e.g. the eventgenerator bosh package does not contain it.
+	go generate ./helpers/apis/generate.go || true
```

Comment:
**Unrelated**

`go generate` will fail with Go `1.22.x` which causes the `Create Bosh Release` to break. 

[Example from CI pipeline](https://github.com/cloudfoundry/app-autoscaler-release/actions/runs/8249543319/job/22562392514#step:9:849): 
```bash
...
+ pushd /github/workspace/extracted1844564904/packages/eventgenerator/autoscaler
  + GOPROXY=off
  + make build-eventgenerator
  /github/workspace/extracted1844564904/packages/eventgenerator/autoscaler /github/workspace/extracted1844564904/packages/eventgenerator
  # Generating OpenAPI clients and servers
  stat ... autoscaler/helpers/apis/generate.go: directory not found
  make: *** [Makefile:40: helpers/apis/scalinghistory] Error 1
  FATAL: Unable to execute command = exit status 2
...
```


Local example: 
```bash
❯ go version
go version go1.22.0 darwin/arm64

❯ go generate /this/dir/doesnt/exist/generate.go
stat /this/dir/doesnt/exist/generate.go: directory not found
```

Doing something like `go generate ./helpers/apis/generate.go || true` ensures that `Create Bosh Release` still works with Go `1.22.x`